source/javascripts/services/_requestService.js.erb

```diff
@@ -543,6 +543,45 @@ import jsyaml from "js-yaml";
 				});
 			};
 
+			requestService.getPipelineConfig = function(requestConfig) {
+				var requestURL;
+
+				switch (requestService.mode) {
+					case "website":
+						requestURL = stringService.stringReplacedWithParameters(
+							"<%= webserver_path(data[:routes][:website][:pipeline_config_get]) %>",
+							{
+								app_slug: requestService.appSlug
+							}
+						);
+
+						break;
+					case "cli":
+						return $q.reject(
+							new Error(
+								"<%= data[:strings][:request_service][:get_pipeline_config][:not_website_mode_error] %>"
+							)
+						);
+				}
+
+				return $q(function(resolve, reject) {
+					$http.get(requestURL, requestConfig).then(
+						function(response) {
+							resolve(response.data);
```

Comment:
Do a snake_case to camelCase transformation with the fields here. So from later on, the fetched data's keys look pretty.

Reply:
We've decided to later come up with a new way for this. For now, this alteration will happen, but later we will use some kind of Typescript type definition for these.